go/multipooler/manager/rpc_consensus.go

```diff
@@ -110,6 +119,13 @@ func (pm *MultiPoolerManager) BeginTerm(ctx context.Context, req *consensusdatap
 
 // ConsensusStatus returns the current status of this node for consensus
 func (pm *MultiPoolerManager) ConsensusStatus(ctx context.Context, req *consensusdatapb.StatusRequest) (*consensusdatapb.StatusResponse, error) {
+	// Acquire the action lock to read consensus state
+	ctx, err := pm.actionLock.Acquire(ctx, "ConsensusStatus")
```

Comment:
Another potential bug caught. Might not matter in this case because an RPC to read consensus state can probably get away with reading stale data, but maybe we want to avoid reading inconsistent data where the consensus term and replication status are out of sync

Reply:
Maybe it'd make sense to add something like `GetInconsistentCurrentTermNumber()` or `GetCurrentTermNumberForMonitoring()` that doesn't require holding the action lock?